File: Constructorio_Net/src/models/RecommendationsResponse.java
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

/**
 * Constructor.io Recommendations Response
 **/
namespace Constructorio_NET
{
    public class RecommendationsResponse {

        [JsonPropertyName("result_id")]
        public String resultId;

        [JsonPropertyName("response")]
        public RecommendationsResponseInner response;

        [JsonPropertyName("request")]
        public Map<String, Object> request;

    }
}

File: Constructorio_NET/src/models/RecommendationsResponse.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;

/**
 * Constructor.io Recommendations Response
 */
namespace Constructorio_NET
{
    public class RecommendationsResponse {

        [JsonProperty("result_id")]
        public String ResultId { get; set; }

        [JsonProperty("response")]
        public RecommendationsResponseInner Response { get; set; }

        [JsonProperty("request")]
        public Dictionary<String, Object> Request { get; set; }

    }
}